app/components/message_attachments/message_attachment.js

```diff
@@ -18,10 +18,12 @@ import AttachmentThumbnail from './attachment_thumbnail';
 import AttachmentTitle from './attachment_title';
 import AttachmentFooter from './attachment_footer';
 
-const STATUS_COLORS = {
+export const STATUS_COLORS = {
```

Comment:
lets move this constant to a file under the constants folder

Reply:
done